src/System.Windows.Forms/src/System/Windows/Forms/DataGridTableStyle.cs

```diff
@@ -996,39 +950,19 @@ public int PreferredRowHeight
             {
                 if (isDefaultTableStyle)
                 {
-                    throw new ArgumentException(string.Format(SR.DataGridDefaultTableSet, "PrefferedRowHeight"));
+                    throw new ArgumentException(string.Format(SR.DataGridDefaultTableSet, nameof(PreferredRowHeight)));
                 }
 
                 if (value < 0)
                 {
-                    throw new ArgumentException(SR.DataGridRowRowHeight);
+                    throw new ArgumentOutOfRangeException(nameof(value), value, SR.DataGridRowRowHeight);
                 }
 
-                prefferedRowHeight = value;
-
-                /*
-                bool needToRedraw = false;
-                DataGridRow[] rows = DataGridRows;
-
-                for (int i = 0; i < DataGridRowsLength; i++)
+                if (prefferedRowHeight != value)
```

Comment:
Looks like an old typo
```suggestion
                if (preferredRowHeight != value)
```
A good opportunity to fix it.